namespace.go

```diff
@@ -139,9 +139,12 @@ func NamespaceWithWebSocket() NamespaceOption {
 // The Azure Environment used can be specified using the name of the Azure Environment set in "AZURE_ENVIRONMENT" var.
 func NamespaceWithEnvironmentBinding(name string) NamespaceOption {
 	return func(ns *Namespace) error {
+		if ns.ResourceURI == "" {
```

Comment:
should check the `ns.Environment.ResourceIdentifiers.ServiceBus` value here? or should we keep that for the new .WithAzureEnvironment(...) behavior?

Reply:
since ns.Environment is public, nothing prevents the user from setting it manually. but the other props are also public, so it would be strange to autopopulate them at that point I think.